src/index.ts

```diff
@@ -272,16 +273,18 @@ class NixInstallerAction extends DetSysAction {
       return;
     }
 
-    if (exitCode !== 0) {
+    if (exitCode === 0) {
+      this.addFact(FACT_HAS_DOCKER, true);
+    } else {
       if (this.forceDockerShim) {
```

Comment:
Do we not want to set `FACT_HAS_DOCKER` when the force flag is on?

Reply:
just because they forced it on doesn't mean they have docker ...

Reply:
Fair enough. Wasn't sure if there were any downstream effects, but it appears no.